---
output:
  pdf_document:
    latex_engine: xelatex
    fig_caption: yes
  word_document: default
mainfont: Palatino
header-includes:
- \usepackage{caption}
- \captionsetup[figure]{labelformat=empty}
- \pagenumbering{gobble}
classoption: a4paper
---


<center>

![
**Figure 1. Rice plot, growth rate, air temperature, and ecological community dynamics**. (**a**) 90 cm $\times$ 90 cm rice plot and Wagner pot alignments. Four rice individuals were grown in each pot (indicated by points in the right panel of **a**). Heights and SAPD of the four red individuals in each plot were measured every day during the monitoring period, and the average values of the four individuals were regarded as representative values for each plot. (**b**) Rice growth rate (cm/day). (**c**) Daily mean air temperature measured at each rice plot. Upper and lower dotted lines indicate daily maximum and minimum air temperature. (**d**) Ecological community compositions and average DNA copy numbers per ml water (reported in Ushio 2022). (**e**) The number of amplicon sequence variants (ASVs) from each water sample (reported in Ushio 2022). For **b**, **c**, and **e**, different colors indicate data from different rice plots.
](00_ReformatFigs/Figure_01.pdf){width=100%}

</center>

<center>

![
**Figure 2. Information transfer between rice growth and ecological community members**. (**a**) An example of the results of the unified information-theoretic causality analysis. Information transfer between air temperature and rice growth rates was quantified. Much higher information transfer was detected from air temperature to rice growth (left panel) compared with the opposite direction (right panel). (**b**) Strength of causal influence from ecological community members to rice growth. Transfer entropy (TE) quantified by the UIC method was used as an index of causal influence. Colors indicate taxa assigned to ASVs. $y$-axis indicates ASV ID. Note that the prefix (e.g., "Fungi_") of the IDs corresponds a major target group of the primer and does not necessarily indicate a taxonomic group assigned to the ASV (see Supplementary file 1). (**c**) eDNA dynamics of putative _Globisporangium_ (Fungi_Taxa00402 in Supplementary file 1). (**d**) eDNA dynamics of _Chironomus kiiensis_ (total DNA copy numbers of five midge ASVs). For **c** and **d**, different colors indicate data from different rice plots.
](00_ReformatFigs/Figure_02.pdf){width=90%}

</center>




<center>

![
**Figure 3. The manipulation experiment performed in 2019 and ecological community compositions before and after the manipulation**. (**a**) Setting of the manipulation experiment in 2019. The number and characters next to each plot indicate the plot number and treatment. The left-top inset shows daily mean air temperature (thick line) and daily maximum and minimum air temperature (dashed lines). The right-bottom inset shows three individuals (red and green points) in each Wagner pot, and the number in each red individual indicate the pot location number. Heights and SPAD of the red individuals were measured. Total eDNA copy numbers of (**b**) putative _Globisporangium_ spp. and (**c**) midge (_Chironomus kiiensis_) in the rice plots. “before” and “after” indicate “from 18 June to 24 June” and “from 25 June to 12 July,” respectively. (**d**) Overall community compositions after the manipulation. Gray, red, and blue indicate CT (control), GN (_Globisporangium nunn_ added), and CK (_Chironomus kiiensis_ removed) treatments, respectively. Each ellipse indicates the overall distribution of each treatment data.
](00_ReformatFigs/Figure_03.pdf){width=100%}

</center>


<center>

![
**Figure 4. Rice growth rate and cumulative growth before and after the manipulation experiment in 2019**. (**a**) Growth rates and (**b**) cumulative growth of the rice individuals in the three treatment (CT = control; GN = _Globisporangium nunn_ added; CK = _Chironomus kiiensis_ removed) before and after the manipulation (cumulative growth was calculated by summing up height growth before the third manipulation or during 10 days after the third manipulation).
](00_ReformatFigs/Figure_04.pdf){width=100%}

</center>



<center>

![
**Figure 5. Differential expression genes analysis**. (**a**) _Globisporangium nunn_-added and (**b**) _Chironomus kiiensis_-removed treatment. Red and blue points indicate significant up- and down-regulated genes, respectively. Upper and lower dashed lines indicate $log_2(1.5)$ and $-log_2(1.5)$, respectively.
](00_ReformatFigs/Figure_05.jpg){width=100%}

</center>


<center>

![
**Figure 6. Examples of differentially expressed genes after the manipulation experiment**. Results of (**a**) Os12g0504050, (**b**) Os11g0184900, (**c**) Os01g0678500, (**d**) Os01g0642200, (**e**) Os08g0162800, and (**f**) Os03g0285700 are presented. $y$-axis represents DESeq2-normalized read counts. Gray, red, and blue indicate CT (control), GN (_Globisporangium nunn_ added), and CK (_Chironomus kiiensis_ removed) treatments, respectively. The gene expressions of the GN treatment in all six genes are statistically clearly different from those of the other two treatments (_P_ < 0.0001) except for GN v.s. CK in **b** (_P_ = 0.0087) and GN v.s. CK **e** (_P_ = 0.00014).
](00_ReformatFigs/Figure_06.pdf){width=100%}

</center>




<center>

![
**Figure 1-figure supplement 1. Monitoring framework of ecological community (as in Ushio 2022)**. (**a**) Locations of the five rice plots in 2017. (**b**) Research framework to generate quantitative environmental DNA time series and network reconstruction. (**c**) Ecological community network reconstructed by nonlinear time series. These figures were reproduced from Ushio (2022).
](00_ReformatFigs/Figure_S01.jpg){width=100%}

</center>


<center>

![
**Figure 1-figure supplement 2. Environmental DNA (eDNA)-based monitoring of ecological communities in 2017**. (**a**) eDNA concentrations of ecological communities. The number on each panel indicates each plot. (**b**) Relative abundance of ecological communities. The data was reported in Ushio (2022).
](00_ReformatFigs/Figure_R01.pdf){width=100%}

</center>



<center>

![
**Figure 3-figure supplement 1. _Globisporangium nunn_ and Midge (_Chironomus kiiensis_) used in the manipulation experiment in 2019**. (**a**) Microscopic image of _Globisporangium nunn_ (photo by M. Tojo). (**b**) _Globisporangium nunn_ was incubated in vermiculite and added to the rice plots. For the control and midge-removed treatments, vermiculite without _Globisporangium nunn_ was added. (**c**) Larvae of midge (_Chironomus kiiensis_) naturally colonized in the rice plots. The larvae were manually removed by a commercial net with $\phi$ 1 mm mesh.
](00_ReformatFigs/Figure_S02.pdf){width=100%}

</center>

<center>

![
**Figure 3-figure supplement 2. Environmental DNA (eDNA)-based monitoring of ecological communities in 2019**. (**a**) eDNA concentrations of ecological communities. The number on each panel and the characters in the parentheis indicates the plot number and treatment, respectively. The blue line in each panel indicates the intensive monitoring period (i.e., daily monitoring from 18 June to 12 July). (**b**) Relative abundance of ecological communities.
](00_ReformatFigs/Figure_R02.pdf){width=100%}

</center>


<center>

![
**Figure 4-figure supplement 1. Rice growth trajectory in 2019.** (**a**) Rice height, (**b**) growth rate, and (**c**) SPAD values of the rice during the growing season in 2019. Points indicate observed values, lines indicate mean trends, and gray lines indicate the field manipulation experiments. Colors indicate the treatments.
](00_ReformatFigs/Figure_S03.pdf){width=100%}

</center>


<center>

![
**Figure 5-figure supplement 1. Differential expression gene (DEG) patterns for each Wagner pot (Pot location-specific analysis).** DEG for (**a**) location 1, (**b**) location 2, and (**c**) location 3 for the control (CT) versus _Globisporangium nunn_-added (GN) treatments. (**d**-**f**) DEG analysis for the control versus _Chironomus kiiensis_-removed (CK) treatments for rice grown at locations 1-3. Red and blue points indicate significant up- and down-regulated genes, respectively. Upper and lower dashed lines indicate $log_2(1.5)$ and $-log_2(1.5)$, respectively.
](00_ReformatFigs/Figure_S04.jpg){width=100%}

</center>


